modules/dxp/apps/portal-workflow/portal-workflow-kaleo-designer-test/src/testFunctional/tests/ProcessBuilderKaleoDesignerReact.testcase

```diff
@@ -84,12 +84,18 @@ definition {
 	}
 
 	@description = "Verify that the user can add Nodes into diagram by drag&drop: Condition"
-	@ignore = "Test Stub"
 	@priority = "5"
 	test CanAddConditionNode {
+		property test.name.skip.portal.instance = "ProcessBuilderKaleoDesignerReact#CanAddConditionNode";
+
+		Workflow.openWorkflowListView();
 
-		// TODO CanAddConditionNode pending implementation
+		Workflow.addNewDefinition(workflowDefinitionTitle = "New Workflow Definition");
 
+		ProcessBuilderKaleoDesignerReact.dragSidebarNodeAndMove(
+			nodeLabel = "Condition",
+			nodePosition = "20,-100",
+			nodeType = "condition");
 	}
```

Comment:
It would be interesting to add a `viewPanelNodePresent` to check the node in the diagram.